proto_test.go

```diff
@@ -8,18 +8,21 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/NYTimes/openapi2proto/protobuf"
+
```

Comment:
Mind grouping `protofbuf` import with its other common `openapi2proto` imports below? `goimports` can also do this for you automatically next time, if you hook it into your tooling.

Reply:
Weird, wonder why goimports put this space...